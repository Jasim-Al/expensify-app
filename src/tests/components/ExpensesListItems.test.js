import React from 'react';
import {shallow} from 'enzyme';
import ExpenseListItems from '../../components/ExpensesListItems';
import expenses from '../fixtures/expenses';

test('should show expense list item', () => {
    let wrapper = shallow(<ExpenseListItems { ...expenses[1] } />);
    expect(wrapper).toMatchSnapshot();
});