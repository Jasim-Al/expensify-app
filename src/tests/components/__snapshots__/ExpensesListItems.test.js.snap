// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should show expense list item 1`] = `
<div>
  <Link
    to="/edit/2"
  >
    <h3>
      burger
    </h3>
  </Link>
  <p>
    $0.70
    -
    28th December, 1969
  </p>
</div>
`;
